.coderabbit.yaml

```
@@ -0,0 +1,4 @@
+reviews:
+  branches:
+    base_branches:
+      - "develop"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)